tests/matrixstore/test_service.py

```diff
@@ -345,6 +347,17 @@ def test_dataset_lifecycle():
     dataset_repo.delete.assert_called_once()
 
 
+def _create_upload_file(filename: str, file: t.IO = None, content_type: str = "") -> UploadFile:
```

Comment:
Need the docstring for this function

Reply:
It doesn't truly justify documenting a unit test function since it's an internal implementation detail. Furthermore, the code is self-explanatory and straightforward. I intend to remove this function in my upcoming PR related to the version upgrade as it will no longer serve a purpose. Thank you nonetheless.